test/builtin/drawing/test_plot_detail_ref/plot-cls-opt.txt

```diff
@@ -0,0 +1,198 @@
+System`Graphics
```

Comment:
If brackets and commas were keep, these files could be evaluated, and would be in proper WL. In that case, `.m` extension would be more appropriated.

Reply:
It's close, but some things like how the numpy arrays underlying NumericArray are presented by using numpy's built-in string conversion that  limits the number of elements printed (and also shows limited precision) aren't WL. The primary intent is human consumption for debugging, but if proves to be too limiting that it's also not readily machine-readable may be we could revisit it and find a way to satisfy both requirements.